CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+#					CONTRIBUTING
```

Comment:
Remove the extra leading spaces. Let the markdown control the style.

Reply:
Done

Reply:
Should be like this (we don't centralize with spaces on markdown code):

```suggestion
# CONTRIBUTING
```